assembly/consumer/lifecycle/docker/Dockerfile

```diff
@@ -33,7 +33,6 @@ ENV JAVA_OPTS -Dcommons.db.schema.update=true \
               -Dcommons.eventbus.url=\${SERVICE_BROKER_ADDR} \
               -Dbroker.host=\${BROKER_ADDR} \
               -Dbroker.port=\${BROKER_PORT} \
-              -Dconsumer.jaxb_context_class_name=org.eclipse.kapua.consumer.lifecycle.LifecycleJAXBContextProvider \
```

Comment:
JAXB is not initialized centrally within the DI context startup - configuration and customization of the jaxb context should be done through appropriate DI context manipulation (nominally: proper module configuration)